Dynamically change the remaining number of characters required.

aymenhamada opened the pull request:
Dynamically change the remaining number of characters required.


Issue : #81

Abhishek-kumar09 commented on src/views/pages/ApplicationsView/ApplicationSteps.js:
```diff
@@ -486,7 +485,7 @@ function FormChallenge({
         errorMessages={[
           'This is a required field',
-          'Text too short. Put in atleast 150 chars.'
+          `Text too short. Put in atleast 150 chars. ${formData.q1 && formData.q1.length ? "Remaining " + (150 - formData.q1.length) + `${formData.q1.length < 149 ?  " characters" : " character"}`: ""} `
```
You should only display:
Text too short. Put in `x` characters more.

aymenhamada replied:
Sure i'll fix that.

Abhishek-kumar09 reviewed (changes requested):
Nicely Done.
Run `npm run format` before pushing your next commit.

aymenhamada commented:
Added a variable containing the minimum number of characters required
`  const MIN_CHAR = 150;`

Also added a function to calculate the real length of a string (Removing white space, so user can't submit 150 wihte space)
`const countChar = string => {
    return string.replace(/\s/g, '').length;
 };`

KeenWarrior commented:
Nice work @aymenhamada  🥳
